pkg/server/module_server_test.go

```diff
@@ -0,0 +1,41 @@
+package server
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/grafana/grafana/pkg/api"
+	"github.com/grafana/grafana/pkg/modules"
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/grafana/grafana/pkg/tests/testsuite"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMain(m *testing.M) {
+	testsuite.Run(m)
+}
+
+func TestWillRunInstrumentationServerWhenTargetHasNoHttpServer(t *testing.T) {
+	cfg := setting.NewCfg()
+	cfg.Target = []string{modules.StorageServer}
+	ms, err := InitializeModuleServer(cfg, Options{}, api.ServerOptions{})
+	require.NoError(t, err)
+
+	go func() {
+		_ = ms.Run()
+	}()
+	time.Sleep(500 * time.Millisecond) // wait for http server to be running
```

Comment:
nit. Is there a way to detect when the httpserver is running ?

Reply:
seems like this may not be enough time for it to start up consistently, based on drone output :(

Reply:
Pretty sure the way I'm setting up the integration tests is wrong. It looks like the db config is different for them in drone than what is shown in the mysql tests docker block.